x-pack/plugins/ml/server/routes/schemas/anomaly_detectors_schema.ts

```diff
@@ -63,14 +63,16 @@ export const anomalyDetectionUpdateJobSchema = {
   groups: schema.maybe(schema.arrayOf(schema.maybe(schema.string()))),
 };
 
+export const analysisConfigSchema = schema.object({
```

Comment:
this could move to the `schemas` folder

Reply:
it was pre-defined already, just moved to a dedicated variable